extensions/bibmanager/src/ListBibmItem.tsx

```diff
@@ -0,0 +1,112 @@
+import { Action, ActionPanel, environment, Icon, List, Toast } from "@raycast/api";
+import { spawn } from "child_process";
+import { join } from "path";
+import { pythonbin } from "./index";
+
+interface Item {
+  title: string;
+  uid: string;
+  link: string;
+  year: number;
+  month: number;
+  content: string;
+  tags: [];
+  authors_tag: [];
+  authors_string: string;
+  pdf: string;
+}
+
+export function ListBibmItem(props: { item: Item; index: number }) {
+  return (
+    <List.Item
+      icon={Icon.Dot}
+      title={props.item.uid ?? "No title"}
+      actions={<Actions item={props.item} />}
+      keywords={[props.item.year.toString(), props.item.title, ...props.item.authors_tag, ...props.item.tags]}
+      detail={<List.Item.Detail markdown={getItemDetail(props.item)} />}
+    />
+  );
+}
+
+function Actions(props: { item: Item }) {
+  return (
+    <ActionPanel title={props.item.title}>
+      <ActionPanel.Section>
+        {props.item.pdf && (
+          <Action.OpenWith path={props.item.pdf} title="Open PDF" shortcut={{ modifiers: [], key: "enter" }} />
+        )}
+        {props.item.link && <Action.OpenInBrowser url={props.item.link} title="Open ADS Link in Browser" />}
+        {!props.item.pdf && (
+          <Action.SubmitForm title="Download PDF" icon={Icon.Download} onSubmit={() => DownloadPDF(props.item.uid)} />
+        )}
+        {props.item.uid && <Action.CopyToClipboard content={props.item.uid} title="Copy bibkey" />}
+        {props.item.content && <Action.CopyToClipboard content={props.item.content} title="Copy bibtex" />}
+        {props.item.link && <Action.CopyToClipboard content={props.item.link} title="Copy ADS Link" />}
+      </ActionPanel.Section>
+    </ActionPanel>
+  );
+}
+
+function getItemDetail(item: Item): string {
+  return `## ${item.title}
+
+${item.authors_string ? "**Authors:** " + item.authors_string : ""}
```

Comment:
FYI, you could use a `List.Item.Detail.Metadata` component to show those: https://developers.raycast.com/api-reference/user-interface/list#list.item.detail.metadata